app/src/main/res/values/strings.xml

```diff
@@ -1,10 +1,12 @@
 <resources>
     <string name="app_name">whereu@</string>
     <string name="action_settings">Settings</string>
+    <string name="verification_code_input_prompt">enter your verification code here</string>
     <string name="verification_input_text">Enter the Verification Number</string>
+    <string name="phone_number_input_prompt">enter your phone number here</string>
     <string name="phone_number_input_text">Enter Your Phone #</string>
```

Comment:
Seems redundant to have text above the box and inside the box saying the same thing. Maybe the one on the inside could be a hint to the # format like `+1XXXXXXXXXX`. Same with the verification thing `XXXXX`